prow/gerrit/client/client.go

```diff
@@ -42,6 +42,8 @@ const (
 	GerritInstance = "gerrit-instance"
 	// GerritRevision is the SHA of current patchset from a gerrit change
 	GerritRevision = "gerrit-revision"
+	// GerritReportLabel is the gerrit label prow will cast vote on, fallback to CodeReview label if unset
+	GerritReportLabel = "gerrit-report-label"
```

Comment:
hmm where are we setting this on jobs? I see where we are reading it.

Reply:
in gerrit prowjob config, it's just a label

Reply:
ahh, interesting. 👍